pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
```

Comment:
configure these here: https://github.com/ClearcodeHQ/pytest-postgresql/blob/master/setup.cfg#L9

Just looked around as to why the coverage isn't being gathered.

Reply:
ah! I was wondering why the coverage failed. Thanks!